pkg/reporter/reporter_test.go

```diff
@@ -130,3 +136,128 @@ func Test_junitReport(t *testing.T) {
 		t.Errorf("Reporting failed")
 	}
 }
+
+func Test_jsonReporterWriter(t *testing.T) {
+	var (
+		report = Report{
+			"good.json",
+			"test/output/example/good.json",
+			true,
+			nil,
+		}
+	)
+	deleteFiles(t)
+
+	bytes, err := os.ReadFile("../../test/output/example/result.json")
+	require.NoError(t, err)
+
+	type args struct {
+		reports    []Report
+		outputDest string
+	}
+	type want struct {
+		data []byte
+		err  assert.ErrorAssertionFunc
+	}
+
+	tests := map[string]struct {
+		args args
+		want want
+	}{
+		"Normal/Output results to a file named 'result.json' (default name)": {
```

Comment:
Please add a test where the string is empty to ensure your response is appropriate.

Reply:
Thank you for pointing out that. I added an empty cases. 20f3c63d82443f8b71a8c8929bb15d50c4563a2f